irohad/multi_sig_transactions/state/mst_state.hpp

```diff
@@ -108,7 +108,7 @@ namespace iroha {
      * @return State that provide difference between left and right states
      * axiom operators:
      * A + B == B + A
-     * A - B == A + (B - A)
+     * A + B == B + (A - B)
```

Comment:
I think that should be done in following way:
1. Use [intersect](https://en.wikipedia.org/wiki/Intersection_(set_theory))/[union](https://en.wikipedia.org/wiki/Union_(set_theory)) for the axiom
2. Axioms related to `operator+` looks strange as a doc for `operator-`

Reply:
1. :+1:
2. it is ok